### ISSUE TITLE
URL's show up in printing

### ISSUE THREAD

**Issue by cfarquharson** [wet-boew/wet-boew#4229]
I just noticed that URLs are displayed in the when printing a page.  While this may be a nice feature for external links that have the full address, it's kinda odd or site links that just go ../../something-en.html.  

Is this the intended design since more links are within the site itself, so it means most links are just ../../anotherthing-en.html.

**Comment by cvrebert** [comment]
FWIW, we removed the CSS responsible for this in Bootstrap v4.

**Comment by LaurentGoderre** [comment]
Good to know! Thanks!

**Comment by hsrudnicki** [comment]
So, are you ready to close this issue or otherwise ?

**Comment by EricDunsworth** [comment]
@hsrudnicki There might still be value in keeping this issue open for the following reasons:
* This issue still currently affects WET 4.0, since it hasn't adopted Bootstrap 4.0 as of yet (still on 3.3.1).
* There seemed to be some interest in adopting or taking inspiration from the script @thomasgohard  found ([Improving Link Display for Print](http://alistapart.com/article/improvingprint)) to potentially list out links at the bottom of printed pages in WET. AFAIK nothing's happened on that front so far.

**Comment by hsrudnicki** [comment]
again, please check if this issue needs to remain open or if otherwise, then please update this issue's status, thanks

**Comment by hsrudnicki** [comment]
Before closing this issue, due to no further discussion or/and status updates, this issue has been labeled "Going Stale" in order to be closed soon, in the meantime please update this issue's status if you think otherwise, thank you

**Comment by EricDunsworth** [comment]
@hsrudnicki Even though this issue has gone stale, it's still a valid issue.

URLs no longer show up when printing in Bootstrap 4.0, but WET 4.0 has yet to adopt it. So the issue is still relevant IMO.

Stale issues aren't fun, but I don't know if it's wise to be closing valid issues just because they've gone stale. For instance, if someone posted an accessibility issue that ended up going stale without a proper decision being rendered, it'd probably be unwise to close it just because of its staleness.

**Comment by hsrudnicki** [comment]
Fine, I just wanted your update in regards to this issue, before labeling this issue as 'Going Stale' you had been notified a couple times to update others about this issue's status.

**Comment by rlambert27** [comment]
Hello,

Just bringing this up again as an important issue because we have had feedback recently on this. Any idea in what version this will be fixed or if WET will update to bootstrap v4?

Roch

**Comment by EricDunsworth** [comment]
@rlambert27 I'll probably send over a PR to remove print URLs from the current incarnation of WET 4.0 sometime this week. It should be a super quick and simple override.

IMO it'd still be cool to implement the script @thomasgohard found into WET (or at least take inspiration from it) in the future.

**Comment by masterbee** [comment]
@EricDunsworth - Thanks for the PR, I am not opposed to the issue as well but want to make sure the CSS solution is a viable address of the issues before we start looking at Bootstrap 4.

**Comment by EricDunsworth** [comment]
@masterbee Just wondering if you have any specific concerns with my PR?

Bootstrap 3 uses CSS to insert the values of ``href`` attributes after every link when printing pages. This isn't default browser behaviour and had so many issues related to partial/incomplete values that it was eventually removed from Bootstrap 4. #7476 overrides WET's Bootstrap 3 behaviour to be in line with Bootstrap 4. It shouldn't have any unexpected side effects.

In this issue, #6686 and wet-boew/theme-gcwu-fegc#180, various WET contributors have expressed concern with what's currently being done. @rlambert27 also mentioned his team recently received (usability?) feedback that conveyed concerns about it.

I like the concept behind what @thomasgohard suggested in https://github.com/wet-boew/wet-boew/issues/4229#issuecomment-56066512, but it'd probably be better off if a dedicated issue was created for that. Since I referred to this issue in my PR's commit, this issue will be automatically closed if/when it gets merged in. So I'll create a separate issue for his suggestion.

**Comment by duboisp** [comment]
@EricDunsworth in your last comment:

> I like the concept behind what @thomasgohard suggested in #4229 (comment), but it'd probably be better off if a dedicated issue was created for that. Since I referred to this issue in my PR's commit, this issue will be automatically closed if/when it gets merged in. So I'll create a separate issue for his suggestion.

Have you created a separate issue for his suggestion? If so what issue number it is?

Thanks

**Comment by EricDunsworth** [comment]
@duboisp Looks like I never got around to it. Sorry :(.

I'll try to create it soon.

**Comment by EricDunsworth** [comment]
@duboisp @shawnthompson Just opened #7860 :)!
